src/include/duckdb/common/serializer/format_deserializer.hpp

```diff
@@ -392,7 +392,8 @@ class FormatDeserializer {
 	}
 
 protected:
-	virtual void SetTag(const char *tag) {
+	virtual void SetTag(field_id_t field_id, const char *tag) {
+		(void)field_id;
```

Comment:
Just copy-pasting my previous comment here...
> Nit: `const`? And do we need `(void)field_id`, as in, does some tidy check complain here if we do not use the parameter? I don't think it does, e.g., for function implementations in duckdb... 🤔

Reply:
Yeah I'm getting "unused parameter" warnings (which turn to failures) otherwise so I (void) it here.

Reply:
Could you solve this the same way as I suggested in my other comment? `SetTag(const char *tag, field_id_t field_id = 0)`, something like that? I am wondering what an `INVALID_FIELD_ID` would look like, though, since this is `uint32_t`...
> Hmm, I am wondering if this is confusing. Would it make sense to reverse the order of parameters to allow for default parameters? But might be something to consider in a different PR.

Reply:
Since field_ids are required I don't want to set a default parameter and have people accidentally leave it out. In theory I could probably replace it with `SetTag(const char *tag, const field_id_t)` but it doesn't matter for the compiler.

Reply:
Would it be alright if I tweaked that in a follow up PR? Since this already covers a lot of surface area and Im kind of under time pressure I really want to get this merged ASAP so I can continue with the next steps.